recipe/meta.yaml

```
@@ -10,13 +10,13 @@ source:
 
 build:
   number: 0
-  skip: true  # [osx or win]
+  skip: true  # [osx]
 
 requirements:
   run:
     - cuda-command-line-tools {{ version }}
     - cuda-visual-tools {{ version }}
-    - gds-tools 1.5.0.59                     # [not aarch64]
+    - gds-tools 1.5.0.59                     # [linux64 or aarch64]
```

Should this just be installed on Linux `x86_64` (since we don't have it on `aarch64` yet)?

```suggestion
    - gds-tools 1.5.0.59                     # [linux64]
```